src/config/media/index.md

```diff
@@ -5,4 +5,6 @@
 To setup audio on your Void Linux system you have to decide if you want to use
 [PulseAudio](./pulseaudio.md) or just [ALSA](./alsa.md).
 
-Some applications require PulseAudio, especially closed source programs.
+Some applications require PulseAudio, especially closed source programs. As an
+alternative to PulseAudio, [PipeWire](./pipewire.md) is a new multimedia
+framework that provides its own implementation of a PulseAudio server.
```

Comment:
Maybe you could be more explicit about pipewire-pulse being a drop in replacement for pulseaudio. Some people might interpret this as "If I want to run closed source applications, I need pulseaudio, pipewire will not work". Replacing the word "alternative" with "drop in replacement" could help a bit.